app/types/IndexPageProps.ts

```diff
@@ -13,4 +13,5 @@ export type DhbVaccineDoseDataForIndexPage = DhbVaccineDoseData & {
 export type IndexPageProps = {
   allDhbsVaccineDoseData: DhbVaccineDoseDataForIndexPage[];
   dataValidAsAtTimeUtc: string;
+  lastCheckedAtTimeUtc: string | null;
```

Comment:
Is there a scenario when it would be null?

Reply:
If we can't find a run? Am I being too defensive??

Reply:
_Actually_ just had a double check why I did this - the underlying type is `string | undefined` - so I'm unsure how to propagate that up to the app?

Reply:
oops i just realised i was thinking of this as the last updated time not last checked. 

It's alg to have this as string | null/undefined then :)

Reply:
yea I'd say so. We can gaurentee it will always be there since we control the files. 
By having `null` the index page now needs to account for what to do when there is a null and there isn't a super obvious answer there too haha. 


But also I'm not fussed about this too much. So whatever you think is best go with it.

Reply:
Yup I gotcha

My thinking was to not show the "last checked at" bit if the values aren't available - I think if it's `null` the whole page will fail (luxon will throw) which is less than ideal, especially since this is very much secondary data.

You're right about us having control - I'll update it to not be null, we can handle the case where it's not if it actually happens haha

Reply:
If we mark it as nullable i.e.  `lastCheckedAtTimeUtc?: string` then we don't need to have `null`  unless we want to be explicit?

Reply:
I've spent 2 years being explicit 😅 - happy to go with the crowd though. Is `?` more common in typescript - that would make sense to me too 🤔 cc @theramis @jishaal

Reply:
🤷

Reply:
> ? more common in typescript - that would make sense to me too 

No hard or fast rule tbh, but generally is the pattern I follow. In this case you can make the call :)

Reply:
TS makes embracing `undefined` much easier 😉

Reply:
Let's follow - will update in the next PR ❤️ 

Thanks team I'm on board with this